schema/me/conversation/message.js

```diff
@@ -40,6 +40,15 @@ export const MessageType = new GraphQLObjectType({
     attachments: {
       type: new GraphQLList(AttachmentType),
     },
+    is_invoice: {
+      description: "True if message is an invoice message",
+      type: GraphQLBoolean,
+      resolve: ({ metadata }) => {
```

Comment:
We have an `isExisty` helper (https://github.com/artsy/metaphysics/blob/888dfee4f4e71ef5edaedd59b0df7f8aa311a798/lib/helpers.js#L26) 

That plus destructuring, this might work:

```
resolve: ({ metadata: { lewitt_invoice_id } }) => isExisty(lewitt_invoice_id)
```